src/App.tsx

```diff
@@ -15,11 +15,11 @@ function App() {
         <Header/>
         <Routes>
           <Route path='/'
-          element={<Home/>}/>
+            element={<Home />} />
```

Comment:
good job fixing the indentation - thanks for catching it

Reply:
The credit goes to the marvellous automatic "Format Selection" of VS code